client/src/partials/fiscal/fiscal.manage.html

```diff
@@ -0,0 +1,90 @@
+<form name="FiscalForm" ng-submit="FiscalManageCtrl.submit(FiscalForm)" autocomplete="off" autocapitalize="off" autocorrect="off" novalidate>
+  <div class="row">
+    <div class="col-xs-12">
+      <p><a ng-href="#/fiscal">
```

Comment:
Since this URL might change, I would use `ui-sref` to change the state back to the `fiscal.list` state.  Does that work?